main.go

```diff
@@ -40,12 +51,12 @@ func main() {
 		return
 	}
 
-	startServer(service)
+	startServer(service, port)
 }
 
-func startServer(zimService *zim.Handler) {
+func startServer(zimService *zim.Handler, port string) {
```

Comment:
Maybe creating a config structure and passing this one instead would make it easier to add more config features later on?

Reply:
I hope that I have read your mind well what you were expecting. :)